lib/jetmon.js

```diff
@@ -488,6 +511,18 @@ function updateStats() {
 		statsdClient.increment( 'stats.workers.free.count', freeWorkers.length );
 		statsdClient.increment( 'stats.workers.working.count', ( arrWorkers.length - freeWorkers.length ) );
 
+		for ( let site_status in checkStats ) {
+			for ( let http_code in checkStats[site_status]['http_code'] ) {
+				statsdClient.increment( `worker.check.${site_status}.code.${http_code}.count`, checkStats[site_status]['http_code'][http_code] );
```

Comment:
Since all these stats are not per worker anymore, does it make sense to name it this way?

Maybe something like `'stats.check.${site_status}.code.${http_code}.count'`

Same applies for lines below

Reply:
I think it's fine to keep them as `worker`, as they come from the workers.